### DISCUSSION TITLE
Sync has stopped working after 2 days of intermittent issues

### DISCUSSION THREAD

**Discussion by adanvdo** [NightscoutFoundation/xDrip#2602] [General]
Sync between master and follower devices has stopped working for me today. I was having intermittent issues over the last couple days and the app prompted me for sync service resets, but as of 3 hours ago it is no longer communicating between my devices. Updated to the latest nightly and updated sync key with no change.

**Comment by RokasBakunas** [comment]
Yes stoped for all.

Add screen

2023-01-04, tr 06:50, adanvdo ***@***.***> rašė:

**Comment by vitalykekukh** [comment]
What is the cause? What are the possible solutions?

**Comment by user-shugar** [comment]
Hi. I'm interesting too how to resolve the problem ? The cause is Nightscout ( for example, now everyone musts to use it for some fee), or some technical issues not related to Nightscout ?

**Comment by tzachi-dar** [comment]
Does the problem still happens? It was solved in my case.

**Reply by vitalykekukh** [reply]
everything is working now

**Reply by motorhunter** [reply]
Problem was solved. I think this issue was related to the Google Cloud Messaging service. Some log messages point to this.

**Comment by adanvdo** [comment]
can confirm, sync is working at the moment

**Comment by Navid200** [comment]
Google messaging was down.  It is back up and running now.

I don't understand the reference to Nightscout though.  
You can still use Nightscout for free.  There are many free options.  This is one example:  https://navid200.github.io/xDrip/docs/Nightscout/GoogleCloud.html
